src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -150,6 +150,28 @@ const Dropdown_Button_Story: ComponentStory<typeof Dropdown> = (args) => {
 
 export const Dropdown_Button = Dropdown_Button_Story.bind({});
 
+const Dropdown_IconButton_Story: ComponentStory<typeof Dropdown> = (args) => {
+  const [visible, setVisibility] = useState(false);
+  return (
+    <Dropdown
+      {...args}
+      onVisibleChange={(isVisible) => setVisibility(isVisible)}
+    >
+      <DefaultButton
```

Comment:
Wouldn't it be more efficient to create another enum and prop called something like `variant` used by `TwoStateButton`? where the variants could be `filter` (current design), `default`, `primary`, `secondary`, `neutral`, and `system ui`. This may extend the current component in a logical way, more closely aligning it with other buttons beyond the filter scenario. It also has toggle behavior built in and may be more appropriate for use by `Dropdown`. The `TwoStateButton` already supports two icons.

Reply:
We can add variants to TwoStateButton but that will take some bandwidth and PR review would again take some time.
Currently, we need this to support an upcoming release.